osu.Game.Rulesets.Catch/Objects/Drawables/DrawableDroplet.cs

```diff
@@ -28,15 +28,24 @@ private void load()
                 _ => new DropletPiece());
         }
 
+        private float startRotation;
+
         protected override void UpdateInitialTransforms()
         {
             base.UpdateInitialTransforms();
 
-            // roughly matches osu-stable
-            float startRotation = RandomSingle(1) * 20;
-            double duration = HitObject.TimePreempt + 2000;
+            // Important to have this in UpdateInitialTransforms() to it is re-triggered by RefreshStateTransforms().
```

Comment:
Arguably, we could move all of these `Random` fetches to `Update` and nuke `RefreshStateTransforms` completely (seems only used in catch)...

Reply:
I think having it where it is right now is preferable. I'm not sure putting that in update would end well.